extensions/wikia/AdEngine/js/video/player/ui/volumeControl.js

```diff
@@ -0,0 +1,58 @@
+/*global define*/
+define('ext.wikia.adEngine.video.player.ui.volumeControl', [
+	'wikia.document',
+	'wikia.log'
+], function (doc, log) {
+	'use strict';
+	var logGroup = 'ext.wikia.adEngine.video.player.ui.volumeControl';
+
+	function createVolumeControl() {
+		var volume = doc.createElement('div'),
+			speaker = doc.createElement('a');
+
+		speaker.className = 'speaker';
+		speaker.appendChild(doc.createElement('span'));
+		volume.className = 'ima-mute-div hidden';
+
+		volume.appendChild(speaker);
+		volume.speaker = speaker;
+		log('volume control is added', log.levels.info, logGroup);
+
+		return volume;
+	}
+
+	function add(video) {
+		var volume = createVolumeControl();
+
+		volume.mute = function () {
+			volume.speaker.classList.add('mute');
+			video.setVolume(0);
+			log('mute', log.levels.info, logGroup);
+		};
+		volume.unmute = function () {
+			volume.speaker.classList.remove('mute');
+			video.setVolume(0.75);
+			log('unmute', log.levels.info, logGroup);
+		};
+
+		volume.addEventListener('click', function(e) {
+			if (video.isMuted()) {
+				volume.unmute();
```

Comment:
but we have logs in both `mute` and `unmute` methods. Do you think it's really needed to add the log here as well? I'm totally for adding more logs but more logs means bigger noise so we have to be careful here